functorch/_src/aot_autograd.py

```diff
@@ -410,8 +426,12 @@ def disable_autocast_manager():
     finally:
         del guard
 
+aot_autograd_compiled_fn_cache = {}
```

Comment:
Global cache here is very suspicious, it means normal cache invalidation by functions going out of scope doesn't work

Reply:
It's gone